airflow/contrib/operators/bigquery_operator.py

```diff
@@ -93,6 +94,8 @@ class BigQueryOperator(BaseOperator):
         expiration as per API specifications. Note that 'field' is not available in
         conjunction with dataset.table$partition.
     :type time_partitioning: dict
+    :param src_fmt_configs: configure optional fields specific to the source format
```

Comment:
I'm confused as to what the dictionary should look like. Based on the tests, is it a [full job resource](https://cloud.google.com/bigquery/docs/reference/v2/jobs)? Why do you say "specific to the source format" when it looks like I can pass in any configuration keys to the job?

I'd expect to see a link such as  https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query which describes what the resource can be.